lldb/test/API/functionalities/data-formatter/data-formatter-stl/libstdcpp/map/TestDataFormatterStdMap.py

```diff
@@ -2,10 +2,14 @@
 Test lldb data formatter subsystem.
 """
 
-
-import lldb
-from lldbsuite.test.decorators import *
-from lldbsuite.test.lldbtest import *
+from lldbsuite.test.decorators import add_test_categories, expectedFailureAll, skip
+from lldbsuite.test.lldbtest import (
+    CURRENT_EXECUTABLE_SET,
+    RUN_SUCCEEDED,
+    STOPPED_DUE_TO_BREAKPOINT,
+    TestBase,
+    line_number,
+)
```

Comment:
I'm no Python expert but I'd say lets we revert these back to keep the style as we do in all the other tests (and because it's drive-by change)